Socket/client/chatapp/src/Component/Common/ChatLists/Channels/ChatChannels/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from "react";
+// @ts-ignore
+import { ArrowDown2TwoToneIcon, ArrowUp2TwoToneIcon } from "@iconbox/iconly";
+// @ts-ignore
+import { StarFillIcon } from "@iconbox/eva";
+
+import Group from "../../../Group";
+
+import { StyledChatChannelsWrapper, StyledHeaderWrapper } from "./style";
+
+interface Props {
+  socket: {
+    on: Function;
+  };
+  count?: string;
+}
+
+const ChatChannels = ({ socket, count }: Props) => {
+  const [users, setUsers] = useState<
+    Array<{
+      id: string;
+      name: string;
+    }>
+  >([]);
+  const [toggleIcon, setToggleIcon] = useState(true);
+
+  useEffect(() => {
+    socket.on("users", (users: any) => {
+      setUsers(users);
+    });
+  }, [socket]);
```

Comment:
We need to close listening on component un-mount. like:
```suggestion
  useEffect(() => {
    const listener = (users: any) => {
      setUsers(users);
    };
    
    socket.on("users", listener);
    
    return () => {
        socket.off("users", listener);
    };
  }, [socket]);
```